python/python/glide/async_commands/core.py

```diff
@@ -6628,3 +6628,86 @@ async def lpos(
             Union[int, List[int], None],
             await self._execute_command(RequestType.LPos, args),
         )
+
+    async def pubsub_channels(
+        self, pattern: Optional[TEncodable] = None
+    ) -> List[bytes]:
+        """
+        Lists the currently active channels.
+        The command is routed to all nodes, and aggregates the response to a single array.
```

Comment:
Routing is not applicable to valkey's `channels` command, so or remove that or describe how it became applicable there.